experimental/telemetry/rudder.go

```diff
@@ -0,0 +1,56 @@
+package telemetry
+
+import rudder "github.com/rudderlabs/analytics-go"
+
+// rudderDataPlaneURL is common for all Mattermost Projects
+const rudderDataPlaneURL = "https://pdat.matterlytics.com"
```

Comment:
The repo and the package is not intended just for Mattermost-authored plugins, but we will be the main consumer of the package. Therefore, it makes sense to make our lives easier. We could change the name of the variable to "defaultRudderDataPlaneURL" or "mattermostRudderDataPlaneURL" to make it more clear.
Nevertheless, any user that want to use their own service instead of mattermost's one can always use the "NewRudderClientWithCredentials".

Reply:
We can also set it up as a variable, so anyone can change that too the same way as rudderWriteKey, but 0/5 on that.